lib/core-classes/DecodersService.ts

```diff
@@ -53,12 +53,15 @@ export class DecodersService {
     this.decoders.set(decoder.deviceModel, decoder);
   }
 
-  registerPayloadController(api: PluginApiDefinition, payloadService: PayloadService) {
+  getPayloadController(payloadService: PayloadService) {
+    const controllers = { actions: {} };
```

Comment:
Here you can use the `ControllerDefinition` type for this variable. (Also nitpicking, the name is not very clear to me)